src/app/cases/case-edit/case-edit-form-data.ts

```diff
@@ -183,6 +183,8 @@ export const FORM_DATA_CASE_EDIT = [
         label: 'captions.Facility',
         validation: ['required'],
         service: 'facilityService',
+        fallbackOptionKey: 'OTHER_FACILITY',
```

Comment:
please add it like this everywhere : fallbackOption: {
  fallbackOptionKey: 'OTHER_FACILITY',
  fallbackOptionValue: 'captions.Facility.OTHER_FACILITY',
}